alzueta.yaml

```diff
@@ -0,0 +1,1849 @@
+- equation: NH3 (+M) <=> H + NH2 (+M)  # Reaction 72
+  type: linear-Burke
+  colliders:
+  - name: M # Ar is reference collider
+    eps: {A: 1, b: 0, Ea: 0}
+    rate-constants:
+    - {P: 0.1 atm, A: 7.23e+29, b: -5.316, Ea: 110862.4}
+    - {P: 1 atm, A: 3.497e+30, b: -5.224, Ea: 111163.3}
+    - {P: 10 atm, A: 1.975e+31, b: -5.155, Ea: 111887.8}
+    - {P: 100 atm, A: 2.689e+31, b: -4.920, Ea: 112778.7}
+  - name: N2
+    eps: {A: 4.49281e+00, b: -9.46265e-02, Ea: -1.10071e+02}
+  - name: O2
+    eps: {A: 1.15210e-01, b: 3.41234e-01, Ea: -3.89210e+02}
+  - name: CO2
+    eps: {A: 9.19583e+00, b: 6.10696e-02, Ea: 9.01088e+01}
+  - name: NH3
+    eps: {A: 1.49004e+01, b: 6.06535e-02, Ea: 2.47652e+02}
+  - name: H2O
+    eps: {A: 1.14560e+01, b: 1.27501e-01, Ea: 3.13959e+02}
+- equation: NH2 + NH2 (+M) <=> N2H4 (+M)  # Reaction 90
+  type: linear-Burke
+  units: {length: m, quantity: kmol, activation-energy: cal/mol}
+  colliders:
+  - name: M # Ar is reference collider
+    eps: {A: 1, b: 0, Ea: 0}
+    rate-constants:
+    - {P: 1.000e-04 atm, A: 1.20024e+25, b: -6.43982e+00, Ea: 2.05356e+03}
+    - {P: 1.000e-03 atm, A: 1.19856e+26, b: -6.43893e+00, Ea: 2.16055e+03}
+    - {P: 1.000e-02 atm, A: 1.80223e+27, b: -6.48573e+00, Ea: 2.49649e+03}
+    - {P: 1.000e-01 atm, A: 1.07660e+28, b: -6.40257e+00, Ea: 3.01349e+03}
+    - {P: 1.000e+00 atm, A: 3.65176e+26, b: -5.65302e+00, Ea: 2.95891e+03}
+    - {P: 1.000e+01 atm, A: 4.51325e+23, b: -4.50139e+00, Ea: 2.58051e+03}
+    - {P: 1.000e+02 atm, A: 2.56240e+20, b: -3.31075e+00, Ea: 2.07400e+03}
+    - {P: 1.000e+03 atm, A: 8.99670e+16, b: -2.12230e+00, Ea: 1.37012e+03}
+    - {P: 1.000e+04 atm, A: 5.28067e+13, b: -1.06166e+00, Ea: 5.64223e+02}
+  - name: N2
+    eps: {A: 1.46848e+01, b: -3.00962e-01, Ea: 1.65146e+02}
+  - name: O2
+    eps: {A: 1.53608e+00, b: -3.90857e-02, Ea: 4.43752e+00}
+  - name: NH3
+    eps: {A: 1.64196e+01, b: -7.29636e-02, Ea: 3.66099e+02}
+  - name: H2O
+    eps: {A: 2.17658e+01, b: -1.14715e-01, Ea: 4.19216e+02}
+- equation: HNO (+M) <=> H + NO (+M)  # Reaction 167
+  type: linear-Burke
+  colliders:
+  - name: M # Ar is reference collider
+    eps: {A: 1, b: 0, Ea: 0}
+    rate-constants:
+    - {P: 0.1 atm, A: 2.0121e+19, b: -3.021, Ea: 47792.0}
+    - {P: 1 atm, A: 1.8259e+20, b: -3.008, Ea: 47880.0}
+    - {P: 10 atm, A: 1.2762e+21, b: -2.959, Ea: 48100.0}
+    - {P: 100 atm, A: 5.6445e+21, b: -2.855, Ea: 48459.0}
+    - {P: 1000 atm, A: 9.7111e+21, b: -2.642, Ea: 48940.0}
+  - name: N2
+    eps: {A: 7.56771e+00, b: -1.96878e-01, Ea: 2.53168e+02}
+  - name: H2O
+    eps: {A: 3.35451e+00, b: 1.33352e-01, Ea: 6.64633e+01}
+
+common_reactions:
+- equation: H + O2 <=> O + OH  # Reaction 1
+  rate-constant: {A: 1.0e+14, b: 0.0, Ea: 1.5286e+04}
+- equation: O + H2 <=> OH + H  # Reaction 2
+  duplicate: true
+  rate-constant: {A: 3.8e+12, b: 0.0, Ea: 7948.0}
+- equation: O + H2 <=> OH + H  # Reaction 3
+  duplicate: true
```

Comment:
The original mechanisms had comments/sources listed for most of the reactions, but they seem to have gotten lost here. I think it would be worth preserving them (with a little bit of cleanup).